### PR TITLE
main/libwacom: run more tests, fix installed python scripts

### PR DESCRIPTION
**Pull request by triallax**

### CODE REVIEW

File: main/libwacom/template.py [hunk 1]
```diff
@@ -10,12 +10,13 @@
     "libgudev-devel",
     "libxml2-devel",
 ]
-checkdepends = ["bash"]
+depends = ["python-libevdev", "python-pyudev", "udev"]
```
Comment (triallax) [inline main/libwacom/template.py:13]: is adding a udev rdep okay here?

**Comment by triallax** [comment]
note: libwacom-update-db will still typically initially fail because it needs /etc/udev/hwdb.d to already exist

File: main/libwacom/template.py (hunk 1 above)
Comment (nekopsykose) [reply main/libwacom/template.py:13]: you don't need it

**Comment by triallax** [comment]
above issue doesn't apply anymore after https://github.com/chimera-linux/cports/commit/649cff96962d40cb6e590f0b8083c7e287287661

File: main/libwacom/template.py [hunk 2]
```diff
@@ -10,12 +10,13 @@
     "libgudev-devel",
     "libxml2-devel",
 ]
-checkdepends = ["bash"]
+depends = ["python-libevdev", "python-pyudev"]
+checkdepends = ["bash", "python-libevdev", "python-pytest", "python-pyudev"]
```
Comment (nekopsykose) [inline main/libwacom/template.py:14]:
```suggestion
depends = ["python-libevdev", "python-pyudev"]
checkdepends = ["bash", "python-pytest"] + depends
```

**Pull request merged**